packages/client/src/api/wargames_api/index.ts

```diff
@@ -700,6 +700,11 @@ const checkReference = (message: MessageCustom, db: ApiWargameDb, details: Messa
   })
 }
 
+export const PostBulkMessages = (dbName: string, archiveMark: MessagePlanning[]) => {
```

Comment:
it looks like there is a bit to complete here too, @lilitkarapetyan  🤣

Reply:
I am workin on it 🤣🤣

Reply:
I am working on it  🤣🤣